app/assets/stylesheets/application.css

```diff
@@ -71,6 +72,8 @@ th h3, th p {
     background-color: #fff;
 }
 
+about.countainer
```

Comment:
So if you could start by doing those couple of things we can knock the rest out when I get over there if you don't mind.